packages/playwright-ct-core/package.json

```diff
@@ -27,7 +27,7 @@
   },
   "dependencies": {
     "playwright-core": "1.51.0-next",
-    "vite": "^5.4.14",
+    "vite": "^5.4.14 || ^6.0.0",
```

Comment:
As per release notes it looks like we can have a lax range there and support both versions: https://vite.dev/guide/migration.html